File: javascript/7kyu/Debug Sum of Digits of a Number.js
// https://www.codewars.com/kata/563d59dd8e47a5ed220000ba

// Debug   function getSumOfDigits that takes positive integer to calculate sum of its digits. Assume that argument is an integer.

function getSumOfDigits(integer) {
    var sum = null;
    var digits =  Math.floor(integer).toString();
    for(var ix = 0; ix < digits.length; ix++) {
      sum += +digits[ix];
    }
    return sum;
  }

File: java/7kyu/Debug Sum of Digits of a Number.java
// https://www.codewars.com/kata/563d59dd8e47a5ed220000ba

// Debug   function getSumOfDigits that takes positive integer to calculate sum of its digits. Assume that argument is an integer.

public class Solution {
  public static int sumOfDigits(int n) {
    int sum = 0;
    while (n > 0) {
      sum += n % 10;
      n /= 10;
    }
    return sum;
  }
}